modal/runner/__init__.py

```diff
@@ -70,7 +72,13 @@ async def get_job(
 
 
 @stub.function(
-    secret=Secret.from_name("ext-api-key"),
+    image=Image.debian_slim(python_version="3.10").pip_install(
+        "sentry-sdk[fastapi]==1.39.1"
+    ),
```

Comment:
ngl not a fan of having to put these deps in `.pip_install` blocks everywhere & duplicating what a pyproject.toml should do :/

i think when @montasaurus adds the `ddtrace` package, it might make sense to consolidate some of this dependency boilerplate with a func that takes in a `modal.Image` and chains in the `.pip_install(` for sentry+ddtrace+other common observability things

at least keep it in a single spot in our python

Reply:
Got exactly that in a PR I'm working on!